### DISCUSSION TITLE
Select unit of measurement

### DISCUSSION THREAD

**Discussion by anddam** [sumatrapdfreader/sumatrapdf#3330] [Q&A] [answered]
Hello,
while displaying `☰ > File > Properties` on an A4 PDF I read

    Page Size: 12.9 x 18.25 in

Is there a way to set the unit of measurement to cm?

**Comment by kjk** [comment]
There's no option to change that.

**Comment by GitHubRulesOK** [comment, marked as answer]
Basically it is system regional thus depends on Windows settings
for some users it should be " Inches for others its cm
![image](https://user-images.githubusercontent.com/12754913/222980561-fed99417-9bec-4947-9233-02491c041730.png)

**Comment by anddam** [comment]
Ah I see thanks.

I am not from US but I keep the system language to US English. 

For those on Windows 11 willing to change the unit of measurements without adding an additional language: go to `Settings > Time & language > Language & region > Region > Administrative language settings > Formats`, there you can fine tune the settings.

 

Now I just need to figure why I scan an A4 page and the resulting PDF document says `Page Size: 32.77 x 46.36 cm` in properties.
